integration-tests/chopsticks/src/network/hydraDx.ts

```diff
@@ -0,0 +1,69 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import * as SpiritnetConfig from './spiritnet.js'
+import { initialBalanceHDX, initialBalanceKILT, toNumber } from '../utils.js'
+
+export const options: SetupOption = {
+	endpoint: process.env.HYDRADX_WS || ['wss://hydradx-rpc.dwellir.com', 'wss://rpc.hydradx.cloud'],
+	db: './db/hydradx.db.sqlite',
+	port: toNumber(process.env.HYDRADX_PORT) || 9001,
+}
+
+export const kiltTokenId = 60
+
+export function setGovernance(addr: string[]) {
+	return {
+		TechnicalCommittee: { Members: addr },
+		Council: { Members: addr },
+	}
+}
+
+export function assignNativeTokensToAccount(addr: string[], balance: bigint = initialBalanceHDX) {
+	return {
+		System: {
+			Account: addr.map((address) => [[address], { providers: 1, data: { free: balance } }]),
+		},
+	}
+}
+
+export function assignKiltTokensToAccount(addr: string[], balance: bigint = initialBalanceKILT) {
```

Comment:
```suggestion
export function assignKiltTokensToAccounts(addr: string[], balance: bigint = initialBalanceKILT) {
```

Reply:
Can you add comments for these functions as well?

Reply:
Updated in: [bec955e](https://github.com/KILTprotocol/kilt-node/pull/614/commits/bec955eb1a2f911de2ef4479826913cacd5e5638)